src/porepy/fracs/meshing.py

```diff
@@ -82,10 +82,6 @@ def subdomains_to_mdg(
     # we are ready to create mortar grids on the interface between subdomains. These
     # will be added to the mixed-dimensional grid.
     create_interfaces(mdg, node_pairs)
```

Comment:
@keileg, note changes in this file.

Reply:
I think I covered this with my comment to `md_grid` , at least that was my intention. Let me know if I failed, or if we should discuss.